src/imageTools/magnify2.js

```diff
@@ -171,6 +177,7 @@ function createMagnificationCanvas (element) {
     // Make sure position is absolute so the canvas can follow the mouse / touch
     magnify.style.position = 'absolute';
     magnify.style.display = 'none';
+    magnify.style.zIndex = '100';
```

Comment:
Don't inline the z-index. Have the implementer include it in their external class styling?